mopidy/mpd/protocol/playback.py

```diff
@@ -397,7 +397,8 @@ def setvol(context, volume):
     - issues ``setvol 50`` without quotes around the argument.
     """
     # NOTE: we use INT as clients can pass in +N etc.
-    context.core.mixer.set_volume(min(max(0, volume), 100))
+    if context.core.mixer.set_volume(min(max(0, volume), 100)).get() is False:
```

Comment:
Maybe even extract `volume = min(max(0, volume), 100)` as a variable of its own.